### PR TITLE
Support the new entropy mechanism for mirage/tls

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

**Comment by samoht** [comment]
This is part of https://github.com/mirage/mirage-dev/pull/52 and  https://github.com/mirleft/ocaml-nocrypto/issues/55

File: lib/conduit_lwt_tls.ml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 open Lwt
 
-let _ = Tls_lwt.rng_init ()
+let _ = Nocrypto_entropy_lwt.initialize ()
```
Comment (hannesm) [inline lib/conduit_lwt_tls.ml]: and we should do this properly, actually... meaning within `lwt_main` loop... not via this racy piece here (yeah, my fault)

**Comment by samoht** [comment]
I'm merging this one for now. @hannesm please feel free to send a better fix!

File: lib/conduit_lwt_tls.ml (hunk 1 above)
Comment (avsm) [inline lib/conduit_lwt_tls.ml]: I created #67 to track this comment.

**Pull request merged**